### PR TITLE
Translated into Simplified Chinese.

### PR DESCRIPTION
**Pull request by Hamachi-geek**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- The app was translated into Simplified Chinese.

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks as always!
I would like to merge this first. Because it will cause a lot of conflict🙇
https://github.com/DroidKaigi/conference-app-2022/pull/387

**Comment by takahirom** [comment]
Can you take a look at the conflict?

File: feature-about/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kittinunf) [inline feature-about/src/main/res/values-zh-rCN/strings.xml:1]: This should be moved to `core.model/commonMain/resources`

Comment (Hamachi-geek) [reply feature-about/src/main/res/values-zh-rCN/strings.xml:1]: In "MR" directly right?

Comment (kittinunf) [reply feature-about/src/main/res/values-zh-rCN/strings.xml]: Yes!

File: feature-announcement/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kittinunf) [inline feature-announcement/src/main/res/values-zh-rCN/strings.xml:1]: This should move into `core-model` resources.

File: feature-contributors/src/main/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kittinunf) [inline feature-contributors/src/main/res/values-zh-rCN/strings.xml:1]: Same here.

Comment (Hamachi-geek) [reply feature-contributors/src/main/res/values-zh-rCN/strings.xml:1]: Sorry. I understood what I have to do right now.

**Comment by kittinunf** [comment]
Or you can rebase on top of #482. It has everything you need except for the translation text itself.

**Review by takahirom — APPROVED** [review]
Thank you for adding the translation!

**Pull request merged**